e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/endpoints/EndpointsEMTest.java

```diff
@@ -0,0 +1,35 @@
+package org.evomaster.e2etests.spring.examples.endpoints;
+
+import com.foo.rest.examples.spring.endpoints.EndpointsController;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.spring.examples.SpringTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class EndpointsEMTest extends SpringTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+
+        SpringTestBase.initClass(new EndpointsController());
+    }
+
+    @Test
+    public void testRunEM() throws Throwable {
+
+        runTestHandlingFlakyAndCompilation(
+                "EndpointsEM",
+                "org.foo.EndpointsEM",
+                500,
+                (args) -> {
+
+                    Solution<RestIndividual> solution = initAndRun(args);
+
+                    assertTrue(solution.getIndividuals().size() >= 1);
+
```

Comment:
here we should  make sure to assert each endpoint has been called.

as RestAssured has no `trace`, shouldn't the test fails when compiled???

Reply:
I added assertHasAtLeastOne() to check for TRACE, OPTIONS and HEAD.
The test case is currently failing since the TRACE command is not generated.

Reply:
thx! i ll make a fix (but unlikely today)